airbyte-integrations/connectors/source-instagram/acceptance-test-config.yml

```diff
@@ -5,7 +5,7 @@ test_strictness_level: high
 acceptance_tests:
   spec:
     tests:
-      - spec_path: "integration_tests/spec.json"
+      - spec_path: "spec.json"
```

Comment:
Does this file actually exist? I've seen @btkcodedev for some reason rename jsons to yamls, and I have no clue why he did that, seems sus.

Reply:
Updated, but this still seems not to have an effect at AT at all.